### PR TITLE
6 user devise auth

### PR DESCRIPTION
**Pull request by LFTPadilla**

### CODE REVIEW

File: app/controllers/articles_controller.rb [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 # Article controller
 class ArticlesController < ApplicationController
-  http_basic_authenticate_with name: 'dhh', password: 'secret', except: %i[index show]
+  #http_basic_authenticate_with name: 'dhh', password: 'secret', except: %i[index show]
```
Comment (MrJuanGaviriaK) [inline app/controllers/articles_controller.rb:5]: We don't need unused comments.

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 # Comments controller
 class CommentsController < ApplicationController
-  http_basic_authenticate_with name: 'dhh', password: 'secret', only: [:destroy]
+  #http_basic_authenticate_with name: 'dhh', password: 'secret', only: [:destroy]
```
Comment (MrJuanGaviriaK) [inline app/controllers/comments_controller.rb:5]: Same here.

File: test/models/user_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (MrJuanGaviriaK) [inline test/models/user_test.rb:1]: We don't need unused files.

File: test/fixtures/users.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (MrJuanGaviriaK) [inline test/fixtures/users.yml:1]: We don't need unused files.

**Review by MrJuanGaviriaK — CHANGES_REQUESTED** [review]
Hi.

I left some comments.

Bests

File: test/models/user_test.rb (hunk 3 above)
Comment (LFTPadilla) [reply test/models/user_test.rb:1]: Ok I will delete the others that are not in use

File: test/fixtures/users.yml (hunk 4 above)
Comment (LFTPadilla) [reply test/fixtures/users.yml:1]: Ok, I will delete it. I didn't know about these types of files

**Review by MrJuanGaviriaK — APPROVED** [review]

**Pull request merged**